### PR TITLE
1169 add non static site initialisation (bootstrap version)

### PR DESCRIPTION
**Pull request by vladb00mer**
as discussed on meeting

### CODE REVIEW

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- jdi-light-examples/src/test/java/io/github/epam/tests/nonstatic/NonStaticSiteTests.java  1
- jdi-light-examples/src/main/java/io/github/com/NonStaticSite.java  2
- jdi-light-examples/src/test/java/io/github/epam/NonStaticTestsInit.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/jdi-light/jdi-light/pullRequest?prid=4509236&bid=15195122)

File: jdi-light-examples/src/test/java/io/github/epam/NonStaticTestsInit.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        nonStaticSite.getHomePage().open();
+        WebSettings.logger.toLog("Non Static site page opened");
+    }
+
+
+    @AfterSuite(alwaysRun = true)
+    public void cleanUp() {
+        WebDriverUtils.killAllSeleniumDrivers();
+    }
+}
```
Comment (RomE-HK) [inline jdi-light-examples/src/test/java/io/github/epam/NonStaticTestsInit.java]: EOF should be at the empty line. Fix it, please

File: jdi-light-examples/src/test/java/io/github/epam/tests/nonstatic/NonStaticSiteTests.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    public void before() {
+        nonStaticSite.shouldBeLoggedIn();
+        nonStaticSite.getHomePage().shouldBeOpened();
+    }
+
+    @Test
+    public void isOpenedTest() {
+        nonStaticSite.getHomePage().isOpened();
+    }
+}
```
Comment (RomE-HK) [inline jdi-light-examples/src/test/java/io/github/epam/tests/nonstatic/NonStaticSiteTests.java]: EOF should be at the empty line. Fix it, please

Comment (vladb00mer) [reply jdi-light-examples/src/test/java/io/github/epam/tests/nonstatic/NonStaticSiteTests.java]: ok done

File: jdi-light-examples/src/test/java/io/github/epam/NonStaticTestsInit.java (hunk 1 above)
Comment (vladb00mer) [reply jdi-light-examples/src/test/java/io/github/epam/NonStaticTestsInit.java]: it is strange but both classes are ends with empty line in my IDE now and I do nothing about that

File: jdi-light-examples/src/main/java/io/github/com/NonStaticSite.java [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    }
+
+    public WebElement getUserIcon() {
+        return userIcon;
+    }
+
+    public LoginForm getLoginForm() {
+        return loginForm;
+    }
+}
```
[the remark this answers is not in the document]
Comment (vladb00mer) [reply jdi-light-examples/src/main/java/io/github/com/NonStaticSite.java]: it is empty line now

File: jdi-light-examples/src/main/java/io/github/com/NonStaticSite.java [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+    @XPath("//*[@ui='label']")
+    private UIElement userName;
+
+    private WebElement userIcon;
+
+    @Css("form")
+    private LoginForm loginForm;
+
+
```
[the remark this answers is not in the document]
Comment (vladb00mer) [reply jdi-light-examples/src/main/java/io/github/com/NonStaticSite.java]: done

File: jdi-light-examples/src/main/java/io/github/com/NonStaticSite.java [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+public class NonStaticSite {
+
+    @Url("/index.html")
+    @Title(value = "Home", validate = CheckTypes.CONTAINS)
+    private HomePage homePage;
+
+    @XPath("//*[@ui='label']")
+    private UIElement userName;
+
+    private WebElement userIcon;
```
[the remark this answers is not in the document]
Comment (vladb00mer) [reply jdi-light-examples/src/main/java/io/github/com/NonStaticSite.java]: this task is not about JDI and all classes has been approved by Alexey and Dmitry prevoiusly

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- jdi-light-examples/src/test/java/io/github/epam/tests/nonstatic/NonStaticSiteTests.java  1
- jdi-light-examples/src/main/java/io/github/com/NonStaticSite.java  2
- jdi-light-examples/src/test/java/io/github/epam/NonStaticTestsInit.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/jdi-light/jdi-light/pullRequest?prid=4509236&bid=15195122)

File: jdi-light-examples/src/main/java/io/github/com/NonStaticSite.java (hunk 5 above)
[the remark this answers is not in the document]
Comment (RomE-HK) [reply jdi-light-examples/src/main/java/io/github/com/NonStaticSite.java]: But we demonstrating that non static site initialization fully supported by JDI. So we should use JDI elements and asserts =)

**Pull request merged**